src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerEngine.java

```diff
@@ -531,19 +526,28 @@ public ActivityProfileState isActive( final AlignmentContext context, final Refe
         }
 
         if (assemblyDebugOutStream != null) {
-            assemblyDebugOutStream.println("\n\n\n\n"+region.getSpan()+"\nNumber of reads in region: " + region.getReads().size() + "     they are:");
-            for (GATKRead read : region.getReads()) {
-                assemblyDebugOutStream.println(read.getName() + "   " + read.convertToSAMRecord(region.getHeader()).getFlags());
+            try {
+                assemblyDebugOutStream.write("\n\n\n\n" + region.getSpan() + "\nNumber of reads in region: " + region.getReads().size() + "     they are:");
+                for (GATKRead read : region.getReads()) {
+                    assemblyDebugOutStream.write(read.getName() + "   " + read.convertToSAMRecord(region.getHeader()).getFlags());
```

Comment:
You've switched from calling `println()` (which writes a newline to the stream) to calling `write()` (which I don't believe writes a newline).

Reply:
Oh yeah right - fixed.